delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/BlazegraphViews.scala

```diff
@@ -506,8 +506,9 @@ object BlazegraphViews {
       evaluate = evaluate(validateP, validateRef),
       tagger = (event: BlazegraphViewEvent) =>
         Set(
+          Event.eventTag,
           moduleType,
-          s"${Projects.moduleType}=${event.project}",
+          Projects.projectTag(event.project),
           s"${Organizations.moduleType}=${event.project.organization}"
```

Comment:
Probably we want the same here: `Organizations.orgTag(event.project.organization)`